code/game/objects/structures/plaques/static_plaques.dm

```diff
@@ -19,6 +19,72 @@
 /obj/structure/plaque/static_plaque/golden/captain
 	name = "The Most Robust Captain Award for Robustness"
 
+//Commission plaques, to give a little backstory to the stations. Commission dates are date of merge (or best approximation, in the case of Meta) + 540 years to convert to SS13 dates.
+
+/obj/structure/plaque/static_plaque/golden/commission
+	name = "Commission Plaque"
+	desc = "Spinward Sector Station SS-13\n'Runtime' Class Outpost\nCommissioned 11/03/2556\n'Dedicated to the Pioneers'"
```

Comment:
Is it out of scope to leave a comment of a date and git commit hash (if you can get one), so you can explain where these dates came from?

Reply:
yeah, I can add those, at least where I can get them